static_src/stores/route_store.js

```diff
@@ -40,6 +42,26 @@ class RouteStore extends BaseStore {
     return !!route.app_guid;
   }
 
+  getRouteURLForApp(app) {
+    const allRoutes = this.getAll();
+    const boundRoutes = allRoutes.filter((route) => route.app_guid === app.guid);
```

Comment:
Probably something we will want to fix down the line, we'll get bit by this eventually if we're not careful about how we're loading the routes.